locksmith/__tests__/operations/userOperations.test.ts

```diff
@@ -0,0 +1,86 @@
+import UserOperations from '../../src/operations/userOperations'
+
+const models = require('../../src/models')
+let User: any = models.User
+let UserReference: any = models.UserReference
+
+describe('User creation', () => {
+  let userCreationDetails = {
+    emailAddress: 'USER@EXAMPLE.COM',
+    publicKey: '0x21cc9c438d9751a3225496f6fd1f1215c7bd5d83',
+    passwordEncryptedPrivateKey: '{"data" : "encryptedPassword"}',
+    recoveryPhrase: 'recoveryPhrase',
+  }
+
+  describe("when a user for the public key doesn't exist", () => {
+    it('should create a user and associated data', async () => {
+      UserReference.create = jest.fn(() => {})
+
+      expect.assertions(1)
+      await UserOperations.createUser(userCreationDetails)
+      expect(UserReference.create).toHaveBeenCalledWith(
+        {
```

Comment:
The logic of the params is actually tested in the test below?
I would rather test that it returns `true`?

Reply:
So that the tests are not identical and we do actually check the output?

Reply:
i'm not sure if these test will provide a lot of value if we are mocking out the implementation of its internals